src/theme/themeTokens.ts

```diff
@@ -0,0 +1,105 @@
+import { PaletteMode } from '@mui/material';
+
+const PRIMARY_LIGHT_THEME = '#222222';
+const PRIMARY_DARK_THEME = '#ffffff';
+
+export const themeTokens = {
```

Comment:
Don't forget about types. It will help you to avoid naming mistakes and suggest more available options according to the type.